doc/releases/release-notes-3.2.rst

```diff
@@ -462,10 +462,454 @@ Build System
+        * :dtcompatible:`gd,gd32-i2c`
+        * :dtcompatible:`gd,gd32-pwm`
+        * :dtcompatible:`gd,gd32-spi`
+        * :dtcompatible:`gd,gd32-syscfg`
+        * :dtcompatible:`gd,gd32-timer`
+        * :dtcompatible:`gd,gd32-usart`
+
+      * Similarly, various GigaDevice peripherals now support the standard
+        ``resets`` property as part of related driver changes to support
+        resetting the peripheral state before initialization via the
+        :dtcompatible:`gd,gd32-rcu` binding:
+
+        * :dtcompatible:`gd,gd32-dac`
+        * :dtcompatible:`gd,gd32-gpio`
+        * :dtcompatible:`gd,gd32-i2c`
+        * :dtcompatible:`gd,gd32-pwm`
+        * :dtcompatible:`gd,gd32-spi`
+        * :dtcompatible:`gd,gd32-usart`
+
+    * Intel SoCs:
+
+      * :dtcompatible:`intel,adsp-tlb`:
+        new ``paddr-size``, ``exec-bit-idx``, ``write-bit-idx`` properties
+      * :dtcompatible:`intel,cavs-shim-clkctl`: new ``wovcro-supported`` property
+      * Removed ``intel,dmic`` binding
+      * Removed ``intel,s1000-pinmux`` binding
+
+    * Nordic SoCs:
+
+      * :dtcompatible:`nordic,nrf-pinctrl`: ``NRF_PSEL_DISCONNECTED`` can be used
+        to disconnect a pin
+      * :dtcompatible:`nordic,nrf-spim`: new ``rx-delay-supported``,
+        ``rx-delay`` properties
+      * :dtcompatible:`nordic,nrf-spim`, :dtcompatible:`nordic,nrf-spi`: new
+         ``overrun-character``, ``max-frequency``, ``memory-region``,
+         ``memory-region-names`` properties
+      * :dtcompatible:`nordic,nrf-uarte`: new ``memory-region``,
+        ``memory-region-names`` properties
+      * Various bindings have had ``foo-pin`` properties deprecated. For
+        example, :dtcompatible:`nordic,nrf-qspi` has a deprecated ``sck-pin``
+        property. Uses of such properties should be replaced with pinctrl
+        equivalents; see :dtcompatible:`nordic,nrfpinctrl`.
+
+    * Nuvoton SoCs:
+
+      * :dtcompatible:`nuvoton,npcx-leakage-io`: new ``lvol-maps`` property
+      * :dtcompatible:`nuvoton,npcx-scfg`: removed ``io_port``, ``io_bit``
+        cells in ``lvol_cells`` specifiers
+      * Removed: ``nuvoton,npcx-lvolctrl-def``, ``nuvoton,npcx-psl-out``,
+        ``nuvoton,npcx-pslctrl-conf``, ``nuvoton,npcx-pslctrl-def``
+      * Added pinctrl support for PSL (Power Switch Logic) pads
+
+    * NXP SoCs:
+
+      * :dtcompatible:`nxp,imx-pwm`: new ``run-in-wait``, ``run-in-debug`` properties
+      * :dtcompatible:`nxp,lpc-spi`: new ``def-char`` property
+      * :dtcompatible:`nxp,lpc-iocon-pinctrl`: new ``nxp,analog-alt-mode`` property
+      * removed deprecated ``nxp,lpc-iap`` binding
+      * :dtcompatible:`nxp,imx-csi`: new ``sensor`` property replacing the
+        ``sensor-label`` property
+      * :dtcompatible:`nxp,imx-lpi2c`: new ``scl-gpios``, ``sda-gpios`` properties
+
+    * STM32 SoCs:
+
+      * :dtcompatible:`st,stm32-adc`: new ``has-vbat-channel`` property
+      * :dtcompatible:`st,stm32-can`: new ``ts-cal-resolution`` property;
```

Comment:
Where do you see this new `ts-cal-resolution` property?

Reply:
Indeed, this should be `st,stm32-temp-cal.yaml`

Reply:
I guess I just made it up. Thanks @henrikbrixandersen ; I removed that.